src/sentry/mediators/param.py

```diff
@@ -127,13 +133,8 @@ def is_required(self):
             return False
         return True
 
-    def _type_match(self, value):
-        if isinstance(self.type, six.string_types):
-            self.type = self._eval_type()
-        return isinstance(value, self.type)
-
-    def _eval_type(self):
-        mod, klass = self.type.rsplit('.', 1)
+    def _eval_string_type(self):
+        mod, klass = self._type.rsplit('.', 1)
```

Comment:
Is there a chance that the root module won't have been imported? Or are they always referencing `sentry`?

Reply:
I think the string form of this will always be referencing `sentry` since it's mostly to avoid circular dependencies. I guess that could theoretically happen with an external dep that we control, but it's probably unlikely.

@MeredithAnya Yea, good call. I'll add some more documentation to these methods.